pages/common/git-count-objects.md

```diff
@@ -0,0 +1,20 @@
+# git count-objects
+
+> Count the number of unpacked objects and their disk consumption.
+> More information: <https://git-scm.com/docs/git-count-objects>.
+
+- Count all objects and display the total disk usage:
+
+`git count-objects`
+
+- Display a count of all objects and their total disk usage, displaying sizes in human readable units:
+
+`git count-objects -H`
```

Comment:
```suggestion
`git count-objects --human-readable`
```

Reply:
Gosh @navarroaxel you're right.  Totally forgot about this one!